### PR TITLE
#RI-4181 - add to tutorial button for recommendations

### PR DESCRIPTION
**Pull request by romansergeenkosofteq**

### CODE REVIEW

File: redisinsight/ui/src/pages/workbench/components/enablement-area/EnablementArea/utils/getFileInfo.ts [hunk 1]
```diff
@@ -130,3 +130,29 @@ export const getParentByManifest = (
 
   return parent ?? null
 }
+
+export const findMarkdownPathByPath = (manifest: IEnablementAreaItem[], markdownPath: string) => {
+  const findPath = (data: IEnablementAreaItem[], mdPath: string, path: number[] = []): null | number[] => {
```
Comment (zalenskiSofteq) [inline redisinsight/ui/src/pages/workbench/components/enablement-area/EnablementArea/utils/getFileInfo.ts:135]: I suggest change `null | number[]` to `Nullable<number[]>`

File: redisinsight/ui/src/slices/app/info.ts [hunk 2]
```diff
@@ -7,7 +7,7 @@ import { GetServerInfoResponse } from 'apiSrc/modules/server/dto/server.dto'
 import { AppDispatch, RootState } from '../store'
 import { RedisResponseEncoding, StateAppInfo } from '../interfaces'
 
-const segmentWriteKey = process.env.SEGMENT_WRITE_KEY || 'SOURCE_WRITE_KEY'
+const segmentWriteKey = 'KSKD2KYXyEC2298VjgAKKDC0pwCgMc1q'
```
Comment (zalenskiSofteq) [inline redisinsight/ui/src/slices/app/info.ts:10]: is it necessary?

Comment (romansergeenkosofteq) [reply redisinsight/ui/src/slices/app/info.ts:10]: removed, force-pushed

File: redisinsight/ui/src/pages/workbench/components/enablement-area/EnablementArea/utils/getFileInfo.ts (hunk 1 above)
Comment (romansergeenkosofteq) [reply redisinsight/ui/src/pages/workbench/components/enablement-area/EnablementArea/utils/getFileInfo.ts:135]: fixed

**Review by zalenskiSofteq — APPROVED** [review]

**Review by zalenskiSofteq — APPROVED** [review]

**Pull request merged**